### ISSUE TITLE
Remove empty catch blocks

### ISSUE THREAD

**Issue by GoodForOneFare** [Shopify/javascript#109]
Remove the empty catch block from code like this:
```js
// Before
try {
  foo();
} catch (undefined) {} finally {
  bar();
}

// After
try {
  foo();
} finally {
  bar();
}

// Before
try {
  foo();
} catch (e) {} finally {
  bar();
}

// After
try {
  foo();
} finally {
  bar();
}
```

**Comment by fandy** [comment]
@lemonmade and I decided it's easier to do this in another code-mod transform instead of Decaf (for the sake of sanity).

**Comment by fandy** [comment]
Fixed in #153
